src/irc/BridgedClient.ts

```diff
@@ -628,6 +628,12 @@ export class BridgedClient extends EventEmitter {
 
     private keepAlive() {
         this.lastActionTs = Date.now();
```

Comment:
Might as well put this below the membership/bot check.

Reply:
We do still use this one for metrics c:, which is a bit misleading I know.